go/libkb/env.go

```diff
@@ -503,8 +516,46 @@ func (e *Env) GetServerURI() string {
 	)
 }
 
+func (e *Env) GetUseRootConfigFile() bool {
+	return e.GetBool(false, e.cmd.GetUseRootConfigFile)
+}
+
+func (e *Env) GetRootRedirectorMount() string {
+	return "/keybase"
```

Comment:
Oh, because Windows is on K:/? Was there some behavior where it uses the next available one if K:/ is taken? Is it possible to get which mount it's on on Windows?

Reply:
Fixed by returning an error if unsupported.